build/ci/vscode-python-nightly-ci.yaml

```
@@ -2,6 +2,8 @@ resources:
 - repo: self
   clean: true
 
+variables: { VSCODE_PYTHON_ROLLING: true }
```

Not sure this needs to be in `{`...`}`. Just put the value on a newline and indent. Why mix `JSON` and `YAML`?

```suggestion
variables: 
  VSCODE_PYTHON_ROLLING: true
```